Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -1159,8 +1163,10 @@ open class NavigationMapView: UIView {
                     let circlesLayer = delegate?.navigationMapView(self,
                                                                    waypointCircleLayerWithIdentifier: waypointCircleLayerIdentifier,
                                                                    sourceIdentifier: waypointSourceIdentifier) ?? defaultWaypointCircleLayer()
-                    
-                    if mapView.mapboxMap.style.layerExists(withId: NavigationMapView.LayerIdentifier.arrowSymbolLayer) {
+                    // Use custom layer position for the waypoint layer.
+                    if (layerPosition != nil) {
```

Comment:
Can you cover these three conditions with unit tests? It'd be important to have them to make sure that we did not break anything.